package.json

```
@@ -8,15 +8,14 @@
     "prewatch": "npm run precompile",
     "watch": "tsc --build --watch tsconfig.json tsconfig.esm.json tsconfig.esnext.json",
     "clean": "tsc --build --clean tsconfig.json tsconfig.esm.json tsconfig.esnext.json",
-    "postinstall": "npm run update-ts-configs && npm run bootstrap",
+    "postinstall": "npm run update-ts-configs",
     "submodule": "git submodule sync --recursive && git submodule update --init --recursive",
-    "version:update": "lerna run version:update",
+    "version:update": "lerna run version",
```

npm native version:

```suggestion
    "version:update": "npm run --ws --if-present version",
```

(I assume lack of --if-present was the error mentioned in sig, worked with this in gh workspace of this PR)

optionally can add ` --` at end if you want to pass flags forward (`npm run version:update --something` -> `npm run version --something` in workspaces) but it's not necessary here

```suggestion
    "version:update": "npm run --ws --if-present version --",
```

---

Yes, it works in a more verbose manner. And worth mentioning that `npm run` doesn't parallel the execution so it is much slower than `lerna run` -- this is the main reason I didn't touch `lerna run`.

---

Good point, tho at speed of version update script parallel's kinda whatever

---

yeah... the speed problem is about test, not version.